ui/framework/src/ui-framework/UiFramework.ts

```diff
@@ -190,6 +199,12 @@ export class UiFramework {
   /** Determines if UiFramework has been initialized */
   public static get initialized(): boolean { return UiFramework._initialized; }
 
+  public static get settingsManager() {

```

Comment:
This should probably be marked as @beta because SettingsManager is beta.

Reply:
added beta tag.